tools/telemetry-generator/package.json

```diff
@@ -36,7 +36,7 @@
 	},
 	"devDependencies": {
 		"@fluidframework/build-common": "^2.0.1",
-		"@fluidframework/eslint-config-fluid": "^4.0.0",
+		"@fluidframework/eslint-config-fluid": "^5.2.0",
```

Comment:
Telemetry-generator is still using npm so this will want a `npm i` to update `package-lock.json`